hadoop-ozone/dev-support/checks/acceptance.sh

```diff
@@ -59,22 +60,21 @@ export OZONE_ACCEPTANCE_SUITE OZONE_ACCEPTANCE_TEST_TYPE
 
 cd "$DIST_DIR/compose" || exit 1
 ./test-all.sh 2>&1 | tee "${REPORT_DIR}/output.log"
-RES=$?
+rc=$?
 
 if [[ "${OZONE_ACCEPTANCE_TEST_TYPE}" == "maven" ]]; then
   pushd result
   source "${DIR}/_mvn_unit_report.sh"
   find . -name junit -print0 | xargs -r -0 rm -frv
   cp -rv * "${REPORT_DIR}"/
   popd
+  ERROR_PATTERN="\[ERROR\]"
 else
   cp -rv result/* "$REPORT_DIR/"
-  if [[ -f "${REPORT_DIR}/log.html" ]]; then
-    cp "$REPORT_DIR/log.html" "$REPORT_DIR/summary.html"
-  fi
-  grep -A1 FAIL "${REPORT_DIR}/output.log" | grep -v '^Output' > "${REPORT_DIR}/summary.txt"
+  grep -A1 FAIL "${REPORT_DIR}/output.log" | grep -v '^Output' > "${REPORT_FILE}"
+  ERROR_PATTERN="FAIL"
```

Comment:
So, it searches for "FAIL"/"ERROR" depending on test type?

Reply:
Yes.  `s3a` tests use Maven to run contract tests from Hadoop, not Robot as usual.